root@192.168.122.138

```diff
@@ -0,0 +1,194 @@
+diff --git a/oci/container.go b/oci/container.go
```

Comment:
Is this filename correct?  I'm seeing "root@192.168.122.138" ?  I'm thinking this is a diff file that might not belong.